Onia/test/condor_3plus1/condor2017B/runMixingRootupler.py

```diff
@@ -32,9 +32,9 @@
 process.rootuple.isMC = cms.bool(False)                 # is mc?
 process.rootuple.onia_mass_cuts = cms.vdouble(0.1,500)    # you may need to adjust this
 process.rootuple.SecondSource.fileNames = cms.untracked.vstring(
-    'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
-    'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
-	 'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
-	 'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
-	 'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE'
+    'root://cmsxrootd.fnal.gov/MIXINPUTPATH/MIXFILEINPUT'
+#    'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
+#    'root://cms-xrd-global.cern.ch/MIXINPUTPATH/INPUTFILE',
```

Comment:
we should keep multiple copies of this line, because sometimes the ZeroBias sample has less events than the MuOnia sample and job will crash.